CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
-cmake_minimum_required(VERSION 3.13)
-cmake_policy(SET CMP0144 NEW)
+cmake_minimum_required(VERSION 3.15)
+if (POLICY CMP0167)
+    # deprecated by definition, TODO: port project to "Modern CMake" https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1
+    cmake_policy(SET CMP0167 OLD) 
```

Comment:
Can we add here small comment about the meaning of this policy? Briefly, in one sentence, about the fact that we use this behavior for find_package(Boost) to load CMake's FindBoost module.